src/frontend/components/UI/Sidebar/components/SidebarTour.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import Tour, { TourStep } from '../../../../components/Tour/Tour'
+import { useTour } from '../../../../state/TourContext'
+
+export const SIDEBAR_TOUR_ID = 'sidebar-tour'
+
+const SidebarTour: React.FC = () => {
+  const { t } = useTranslation()
+  const { isTourActive } = useTour()
+
+  const steps: TourStep[] = [
+    {
+      element: '[data-tour="sidebar-menu"]',
+      intro: t(
+        'tour.sidebar.welcome.intro',
+        'On the Sidebar you will find all navigation options to explore the app.'
+      ),
+      title: t('tour.sidebar.welcome.title', 'Sidebar Navigation')
+    },
+    {
+      element: '[data-tour="sidebar-library"]',
+      intro: t(
+        'tour.sidebar.library',
+        'Access your game library from different stores in one place.'
+      ),
+      position: 'right'
```

Comment:
I think we should check if the current language is RTL and use `'left'` here in those cases

Reply:
done